janus_messages/src/time.rs

```diff
@@ -1,6 +1,6 @@
 //! Utilities for timestamps and durations.
```

Comment:
FWIW: Brandon's message refactor in #534 means that we don't have to move this module from `janus_core` to `janus_messages` on `main`.